src/routes/spirit-board/growth.svelte

```diff
@@ -23,29 +71,73 @@
 				{/if}
 				</span></h6>
 			{#if spiritBoard.growth.isVisible}
+				{#if !spiritBoard.growth.useGrowthSets}
 					<div class="field">
 						<div class="control">
-							<button class="button is-primary is-light" on:click={addGrowthSet}>Add Growth Set (ie. Lure)</button>
-						</div>
-					</div>
-					<div class="field">
-						<div class="control">
-							<button class="button is-primary is-light" on:click={addGrowthSet}>Add Growth Group</button>
+							<button class="button is-primary is-light row-button" on:click={useGrowthSets}>Use Growth Sets</button>
 						</div>
 					</div>
+				{/if}
 				{#each spiritBoard.growth.growthSets as growthSet, i (growthSet.id)}
-					<div class="field">
-						<label class="label" for="spiritGrowthInput">Growth</label>
-						<div class="control">
-							<input
-								id="spiritGrowthInput"
-								class="input"
-								type="text"
-								bind:value={spiritBoard.growth.name}
-							/>
+					<div class="growth-set">
+						{#if spiritBoard.growth.useGrowthSets}
+							<div class="growth-set-title">
+								<div class="label">Growth Set
+								</div>
+							</div>
+						{/if}
+						<div class="growth-set-info">
+							{#if spiritBoard.growth.useGrowthSets}
+								<div class="control">
+									<input
+										id="spiritGrowthInput"
+										class="input"
+										type="text"
+										placeholder="Growth Set Action ie. (PICK ONE OF)"
+										bind:value={growthSet.choiceText}
+									/>
+								</div>
+							{/if}
+							{#each growthSet.growthGroups as growthGroup, j (growthGroup.id)}
+								<div class="growth-group">
+									<div class="growth-group-title">
+										<div class="label">Growth Group
+										</div>
+									</div>
+									<div class="growth-group-info">
+										{#each growthGroup.growthActions as growthAction, k (growthAction.id)}
+											<div class="control">
+												<input
+													id="spiritGrowthInput" 
+													class="input"
+													type="text"
+													placeholder="Growth Action"
+													bind:value={growthAction.effect}
+												/> <!-- Eric, does the bind syntax look right? -->
+											</div>
+										{/each}
+										<div class="control">
+											<button class="button is-primary is-light row-button" on:click={addGrowthAction(i,j)}>Add Growth Action</button> <!-- Could I just pass the growthgroup as growthGroup instead of the indexes? -->
```

Comment:
I think you could replace the second argument `j` with `growthGroup` instead if you think that reads better. You need to keep `i` because in the end you need to know which `growthGroup` in `spiritBoard.growth.growthSets[setIndex]` you're adding to. I don't think the `spiritBoard` sees modifications to a `growthGroup` being passed.